Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-contests.tsx

```diff
@@ -90,29 +113,52 @@ const ContestsProvider = ({ children }: IContestsProviderProps) => {
 
     const { state: { strategies, isLoaded: strategiesAreLoaded } } = useContestStrategyFilters();
     const { state: { categories, isLoaded: categoriesAreLoaded } } = useContestCategories();
+    
+    const { actions: { clearBreadcrumb } } = useCategoriesBreadcrumbs();
 
     const possibleFilters = useMemo(
         () => strategiesAreLoaded && categoriesAreLoaded
             ? generateStatusFilters()
                 .concat(generateCategoryFilters(categories))
-                .concat(generateStrategyFilters(strategies))
-            : [],
+                .concat(generateStrategyFilters(strategies)) as IFilter[]
+            : [] as IFilter[],
         [ categories, categoriesAreLoaded, strategies, strategiesAreLoaded ],
     );
 
+    const possibleSorting = useMemo(
+        () => generateSortingStrategy() as unknown as ISort[],
```

Comment:
Is this cast 'as unknown' needed?

Reply:
Unfortunately we need to cast it to unknown in order to be able to cast it to ISort[]. Otherwise there is an error saying the types do not overlap sufficiently with each other when we do not use unknown.